libs/core/src/components/pds-accordion/pds-accordion.scss

```diff
@@ -89,6 +97,7 @@ summary {
 }
 
 .pds-accordion__body {
+  animation: slide-down var(--animation-transform-timing);
```

Comment:
Seeing 2 things and am not 100% sure why there's a difference.

1. Chrome doesn't seem to animate. Not sure if it's not animating or if 200ms is just super fast.

2. Safari now animates (and feels much slower than Chrome), but I see an overlap with the trigger and the sub-items.
![Screenshot 2024-04-25 at 9 14 40 AM](https://github.com/Kajabi/pine/assets/1175111/5ab6769e-4e8f-4e1b-8caa-b237f4dd01af)

If I move the `.pds-accordian__body` styles from line 99 into line 57 they both behave the same, but the overlap issue happens. This is in storybook, so not 100% sure if shows up in KP properly.